### DISCUSSION TITLE
Is it possible to change the plotting algorithm so that it will reduce the wearing out of SSDs?

### DISCUSSION THREAD

**Discussion by donaldmcgillavry** [Chia-Network/chia-blockchain#4009] [Ideas]
Dear Chia developers, could you please tell if it is possible to improve the plotting algorithm somehow to reduce the speed/amount of wearing out the SSDs during plotting?
Or is it just impossible with the chosen consensus algorithm?
If it is at least theoretically possible — do you have plans for doing it?
Because right now the most NON-ecological/NON-"green" thing about Chia is that it will (and is already beginning to) generate tons of discarded worn-out SSDs that were killed quickly and for no obvious purpose.

**Comment by Muffexx** [comment]
The next NON-ecological/NON-"green" thing about Chia are the current timeouts that require the drives to spin all the time.
If the response time thresholds would be increased the drives could sleep while there are no eligible plots in your farm.
Another side-effect of higher thresholds would be less frustrated farmers using (multiple) NAS.
